docs/guide/pt-BR/usage-standalone.md

```diff
@@ -0,0 +1,70 @@
+# Standalone usage
+
+## With configuration file
+
+1. Copy configuration file `./vendor/yiisoft/db-migration/bin/yii-db-migration.php` to root folder of your project:
+
+    ```shell
+    cp ./vendor/yiisoft/db-migration/bin/yii-db-migration.php ./yii-db-migration.php
+    ```
+
+2. Define DB connection in configuration file (see
+   [Yii DB documentation](https://github.com/yiisoft/db/blob/master/docs/en/README.md#create-connection)).
+   For example, MySQL connection:
+
+    ```php
```

Comment:
Extra padding

Reply:
just for the links to exist and function for now. When I start translating this package I will make a new copy of the English original to ensure I have the latest updates. For now, ignore the 'pt-BR' folder, please.